src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -60,12 +61,30 @@
           :error-handler (flash-message/default-error-handler :top description)})
    {}))
 
-(defn- to-create-catalogue-item []
+(rf/reg-event-db
+ ::set-selected-items!
```

Comment:
nit: I checked the codebase for other similar cases and it looks like it's not customary in our codebase to use exclamation mark with these reg-event-db setters, so you might want to align this with that convention.

Reply:
Makes sense